components/Menu/District.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FunctionComponent } from 'react';
```

Comment:
Nope ... ditch this entire file.
The whole purpose of the playground is so that u can add this kind of hardcoded test data to it for testing your widget.

Reply:
If it has a different purpose, don't name it `District`.
This menu doesn't need to be only used for rendering spotlight data ... build it to be ignorant of the context of its data... its purpose is displaying a nice menu.

Reply:
Review the existing SpotlightMenu component to see what I mean ... and it's usage in the playground.
Actually, you're going to have to rename the old SpotlightMenu to MMenu, then name your main component `SpotlightMenu`

Reply:
Apply this to all your other components. From what I see, most of them are going to go ...